src/app/components/client/Chart.module.scss

```diff
@@ -74,11 +75,25 @@
 
   .legend {
     align-self: center;
+    flex-direction: column;
+    display: flex;
+    gap: $spacing-sm;
 
     @media screen and (min-width: $screen-lg) {
       height: auto;
-      display: flex;
       align-items: center;
+      max-width: 200px; // width of legend and prompt
+    }
```

Comment:
Nit: Instead of setting the `max-width` here we could also remove [flex: 1;](https://github.com/mozilla/blurts-server/pull/3260/files#diff-d4128bf7199e93b8e25ef807744020570dc0f56ec1fdba399df71f01c141fdb7R53) from `.chart` and then also just the entire `@media` query here.

Reply:
Hmm I'm not too familiar with the effect here. wdyt @codemist

Reply:
@mansaj We can also leave it as is. I’m making small adjustments to the layout to fit the needs of the dashboard fixed tab layout.